### ISSUE TITLE
UnIDed rare rings getting auto-listed with normal/white rings

### ISSUE THREAD

**Issue by CapitalF** [Novynn/acquisitionplus#43]
I often sell white/normal rings of item level 75+, but I am having a problem where my Rare unIDed rings are being auto-listed at the same price.

I have an "inbox" tab where I temporarily store such items until I can organize them later, and I can ignore that tab by name as a workaround, but it would be nice if this issue was resolved.

This problem likely affects more than just rings.

**Comment by Novynn** [comment]
You're correct, I have to update the item system with item IDs rather than a hash.
